tests/tests/app/integration/test_app_error_visibility.py

```diff
@@ -10,20 +10,6 @@
 ERROR_PYTHON_ERROR = "[ERROR - POST raise_python_error] raise_python_error"
 
 
-@pytest.mark.parametrize("app_handler", [TestApp], indirect=True)
```

Comment:
What's the thinking behind removing these? 

These help us achieve a guarantee that the python-side of the Client -> Engine -> Proxy -> App/Plugin stack is reporting errors in a standardized way.